addons/pos_restaurant/static/src/overrides/models/pos_store.js

```diff
@@ -60,8 +60,12 @@ patch(PosStore.prototype, {
             const table_ids = response.map((floor) => floor.raw.table_ids).flat();
             await this.data.read("restaurant.table", table_ids);
         }
-
         const tableByIds = this.models["restaurant.table"].getAllBy("id");
+        await this.loadServerOrders([
+            ["config_id", "in", [...this.config.raw.trusted_config_ids, this.config.id]],
+            ["table_id", "in", this.models["restaurant.table"].getAll().map((t) => t.id)],
+            ["session_id", "=", odoo.pos_session_id],
```

Comment:
```suggestion
```

I don't think you need the session id, otherwise it will conflict with the `[config_id in ...]` domain.

Reply:
you're right.